src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -153,7 +153,7 @@ public static AddressBook getTypicalAddressBook() {
 
 
     public static List<Person> getTypicalPersons() {
-        return new ArrayList<>(Arrays.asList(ALICE_SUPPLIER.get(), BENSON_SUPPLIER.get(), CARL_SUPPLIER.get(),
+        return new ArrayList<>(Arrays.asList((Person) ALICE_SUPPLIER.get(), BENSON_SUPPLIER.get(), CARL_SUPPLIER.get(),
                 DANIEL_SUPPLIER.get(), ELLE_SUPPLIER.get(), FIONA_SUPPLIER.get(), GEORGE_SUPPLIER.get()));
```

Comment:
⛏️  Nit: Could we [avoid complicated expressions](https://nus-cs2103-ay2526s1.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-complicated-expressions) by using `List#of` instead?

```suggestion
        return List.of(ALICE_SUPPLIER.get(), BENSON_SUPPLIER.get(), CARL_SUPPLIER.get(),
                DANIEL_SUPPLIER.get(), ELLE_SUPPLIER.get(), FIONA_SUPPLIER.get(), GEORGE_SUPPLIER.get());
```